### PR TITLE
Two small fixes found onboarding RWKV

### PR DESCRIPTION
**Pull request by voznesenskym**

### CODE REVIEW

File: torchdynamo/variables/nn_module.py [hunk 1]
```diff
@@ -115,11 +115,15 @@ def var_getattr(self, tx, name):
 
         if name in base_dict:
             subobj = base_dict[name]
-        elif name in base_dict["_modules"] and name not in all_class_attribute_names:
```
Comment (voznesenskym) [inline torchdynamo/variables/nn_module.py:118]: JIT-ed NN Module do not have these attr, so instead of a keyerror we shuffle on to let inspect. getattr_static handle it (or fail with a better message).

File: torchinductor/lowering.py [hunk 2]
```diff
@@ -1321,7 +1321,9 @@ def full_like(x, fill_value, **kwargs):
 
 def tensor_constructor(fill_value):
     # torch.zeros, torch.ones, etc
-    def inner(*size, dtype=None, device=None, layout=0, pin_memory=False):
+    def inner(
+        *size, dtype=None, device=None, layout=0, pin_memory=False, memory_format=None
```
Comment (voznesenskym) [inline torchinductor/lowering.py:1325]: Unused arg, but allows construction of tensors like `y = torch.empty((B, T, C), device='cuda', memory_format=torch.contiguous_format)`

**Review by jansel — APPROVED** [review]

File: torchinductor/lowering.py (hunk 2 above)
Comment (ngimel) [reply torchinductor/lowering.py:1325]: We should probably check that it's `contiguous`, otherwise we'd silently ignore things like `channels_last`?

Comment (voznesenskym) [reply torchinductor/lowering.py:1325]: We don't use the value, do you want me to assert that it is contiguous_format?

Comment (ngimel) [reply torchinductor/lowering.py:1325]: Yes, assert that it's either None or contiguous_format (similarly to how we do with pin_memory and layout)
